packages/theme-chalk/src/date-picker/year-table.scss

```diff
@@ -13,7 +13,7 @@
   td {
     width: 68px;
     text-align: center;
-    padding: 8px 0px;
+    padding: 20px 0px;
```

Comment:
Why change the style here?

Reply:
> Why change the style here?

Oh , not necessary indeed, have been reverted